tests/unit/-private/properties/fillable-test.js

```diff
@@ -13,62 +13,77 @@ moduleForProperty('fillable', function(test) {
       foo: fillable(expectedSelector)
     });
 
-    this.adapter.fillIn((actualSelector, actualContext, options, actualContent) => {
-      assert.equal(actualSelector, expectedSelector);
-      assert.equal(actualContent, expectedText);
-    });
-
     this.adapter.createTemplate(this, page, '<input>');
 
     page.foo(expectedText);
 
-    return this.adapter.wait();
+    return this.adapter.andThen(() => {
+      assert.equal(this.adapter.$(expectedSelector).val(), expectedText);
+    });
   });
 
-  test('looks for inputs or contenteditables with data-test="clue" attributes', function(assert) {
-    let expectedText = 'dummy text';
-    let clue = 'clue';
-    let page;
-
-    page = create({
-      scope: '.scope',
-
-      foo: fillable()
+  const targetAttributes = ['data-test', 'aria-label', 'placeholder', 'name', 'id'];
+
+  const formControlTemplates = [
+    '<input data-test="clue" />',
+    '<input aria-label="clue" />',
+    '<input placeholder="clue" />',
+    '<input name="clue" />',
+    '<input id="clue" />',
+
+    '<textarea data-test="clue"></textarea>',
+    '<textarea aria-label="clue"></textarea>',
+    '<textarea placeholder="clue"></textarea>',
+    '<textarea name="clue"></textarea>',
+    '<textarea id="clue"></textarea>',
+
+    '<select data-test="clue"><option></option><option>dummy text</option></select>',
+    '<select aria-label="clue"><option></option><option>dummy text</option></select>',
+    '<select placeholder="clue"><option></option><option>dummy text</option></select>',
+    '<select name="clue"><option></option><option>dummy text</option></select>',
+    '<select id="clue"><option></option><option>dummy text</option></select>',
+  ];
+
+  formControlTemplates.forEach(template => {
+    let gtPos = template.indexOf('=');
+    let name = template.substr(1, gtPos - 1);
+    const [tagName, attrName] = name.split(' ');
+
+    test(`looks for ${tagName} with ${attrName}`, function(assert) {
```

Comment:
need to figure out how to simplify `[contenteditable]`, `input`, `textarea`, `select` tests